internal/pkg/template/workload.go

```diff
@@ -356,6 +356,7 @@ type WorkloadOpts struct {
 	Aliases                  []string
 	Tags                     map[string]string        // Used by App Runner workloads to tag App Runner service resources
 	NestedStack              *WorkloadNestedStackOpts // Outputs from nested stacks such as the addons stack.
+	AddonsExtraParams        string                   // Additional user defined Parameters for the addons stack.
```

Comment:
Should this be `*string`?

Reply:
I don't think it matters in this scenario 🤔 . It makes my life a bit easier if it's a string because the `Parameters` method's signature returns an empty string instead of nil if there is no extra params defined:
https://github.com/aws/copilot-cli/pull/2978/files#diff-cc1f6de0ed3926d7713d70861fd635ec7ff5fccaa08d0d7d308411990761bf7cR110